sdk/appconfiguration/app-configuration/samples/v1/typescript/src/secretReference.ts

```diff
@@ -101,8 +102,9 @@ async function setup(key: string) {
 
 async function createConfigSetting(key: string, secretId: string) {
   // Set the following environment variable or edit the value on the following line.
-  const connectionString = process.env["APPCONFIG_CONNECTION_STRING"] || "<connection string>";
-  const appConfigClient = new AppConfigurationClient(connectionString);
+  const endpoint = process.env["AZ_CONFIG_ENDPOINT"] || "<connection string>";
```

Comment:
Does the placeholder need to change too (same for the other samples)?

```suggestion
  const endpoint = process.env["AZ_CONFIG_ENDPOINT"] || "<endpoint>";
```

Reply:
Good catch 👍